.github/dependabot.yaml

```diff
@@ -17,6 +17,8 @@ updates:
       - dependencies
     reviewers:
       - jtakalai
+    assignees:
```

Comment:
This PR was forgotten earlier and also repo was forgotten and looks like a mess. CI can't pass.

Change reviewer to kare.